.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/code/go"
+    schedule:
+      interval: "daily"
+    labels:
+      - dependencies
+      - go
+  	open-pull-requests-limit: 10
```

Comment:
Is indentation correct here? (it seems to be at the same level as labels items).

Reply:
```suggestion
    open-pull-requests-limit: 10
```